wicketd/src/update_tracker.rs

```diff
@@ -830,6 +830,23 @@ impl Drop for SetTrueOnDrop {
     }
 }
 
+#[derive(Clone)]
```

Comment:
This is almost exactly like the `OxideSled` enum we added in sled-hardware/types/src/lib.rs; I wonder if we can combine them somehow.

Reply:
Ah yep I forgot to convert that over